src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -37,17 +37,23 @@ static void SendFullAddressInfoToListener(TracerListener* listener,
 
   FullAddressInfo address_info;
   address_info.set_absolute_address(libunwindstack_frame.pc);
-  address_info.set_module_name(libunwindstack_frame.map_name);
+
+  // Careful: unwindstack::FrameData::map_info might contain nullptr.
```

Comment:
I guess we also want to add tests for the case that map info is nullptr. Maybe in a different PR?

Reply:
You're right, it didn't seem part of the merge. https://github.com/google/orbit/pull/3637 will add a test after this change.